Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -0,0 +1,347 @@
+﻿using System;
+using System.Globalization;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public class BalanceStats
+    {
+        private const string NotAvailable = "N/A.";
+        private const decimal Epsilon = 0.00001m;
+
+        /// <summary>
+        /// Return name and balance(current) of person who had the biggest historic balance.
+        /// </summary>
+        public static string FindHighestBalanceEver(Person[] people)
+        {
+            if (people == null)
+            {
+                return NotAvailable;
+            }
+
+            var largestHistoricBalance = GetLargestHistoricBalance(people);
+            var largestHistoricBalanceNames = GetLargestHistoricBalanceNames(people, largestHistoricBalance);
+
+            CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;
+            var largestHistoricBalanceNoComma = RemoveComma($"{largestHistoricBalance:C0}");
+
+            return $"{FixPlural(largestHistoricBalanceNames)} had the most money ever. {largestHistoricBalanceNoComma}.";
+        }
+
+        /// <summary>
+        /// Return name and loss of a person with a biggest loss (balance change negative).
+        /// </summary>
+        public static string FindPersonWithBiggestLoss(Person[] people)
+        {
+            if (people == null)
+            {
+                return NotAvailable;
+            }
+
+            var isBiggestLossValid = TryGetBiggestLoss(people, out var biggestLoss);
+
+            if (!isBiggestLossValid)
+            {
+                return NotAvailable;
+            }
+
+            var biggestLossNames = GetBiggestLossNames(people, biggestLoss);
+
+            CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;
+            return $"{biggestLossNames} lost the most money. -{biggestLoss:C0}.";
+        }
+
+        /// <summary>
+        /// Return name and current money of the richest person.
+        /// </summary>
+        public static string FindRichestPerson(Person[] people)
+        {
+            if (people == null)
+            {
+                return NotAvailable;
+            }
+
+            var largestCurrentBalance = GetLargestCurrentBalance(people);
+            var largestCurrentBalancesNames = GetLargestCurrentNames(people, largestCurrentBalance);
+
+            CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;
+            var word1 = largestCurrentBalancesNames.ToString().Contains(", ") ? "are" : "is";
+            var word2 = largestCurrentBalancesNames.ToString().Contains(", ") ? "people" : "person";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [There's no need to call 'ToString()' on a string.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5116146)